overlord/hookstate/ctlcmd/services_test.go

```diff
@@ -95,6 +130,8 @@ func (s *servicectlSuite) SetUpTest(c *C) {
 	s.st.Lock()
 	defer s.st.Unlock()
 
+	snapstate.ReplaceStore(s.st, &s.fakeStore)
```

Comment:
Does this need to be undone?

Reply:
I don't think we need to. None of the tests we currently have restore it after calling ReplaceStore.

Reply:
the store is attached to the cache in the state, each test creates a new state, so no